test/integration/targets/nxos_ip_interface/tests/cli/sanity.yaml

```diff
@@ -0,0 +1,79 @@
+---
+- debug: msg="START TRANSPORT:CLI nxos_ip_interface sanity test"
+
+- set_fact: testint1="{{ nxos_int1 }}"
+- set_fact: testint2="{{ nxos_int2 }}"
+
+- name: "Put interface {{testint1}} into default state"
+  nxos_config: &intdefault1
+    lines:
+      - "default interface {{ testint1 }}"
+    provider: "{{ cli }}"
+  ignore_errors: yes
+
+- name: "Put interface {{testint2}} into default state"
+  nxos_config: &intdefault2
+    lines:
+      - "default interface {{ testint2 }}"
+    provider: "{{ cli }}"
+  ignore_errors: yes
+
+- name: "Make {{testint1}} a layer3 interface"
+  nxos_interface: &l3int1
+    interface: "{{ testint1 }}"
+    mode: layer3
+    description: 'Configured by Ansible - Layer3'
+    admin_state: 'up'
+    state: present
+    provider: "{{ cli }}"
+
+- name: "Make {{testint2}} a layer3 interface"
+  nxos_interface: &l3int2
+    interface: "{{ testint2 }}"
+    mode: layer3
+    description: 'Configured by Ansible - Layer3'
+    admin_state: 'up'
+    state: present
+    provider: "{{ cli }}"
+
+- name: Ensure ipv4 address is configured
+  nxos_ip_interface: &ipv4
+    interface: "{{ testint1 }}"
+    version: v4
+    state: present
+    addr: 20.20.20.20
+    mask: 24
+    provider: "{{ cli }}"
+  register: result
+
+- assert: &true
+    that:
+      - "result.changed == true"
```

Comment:
@Qalthos ... the tests are fine. It's the platform.

Here are the results on the n9k vms and hardware:

root@agent-lab15-pm:~/agents-ci/ansible/test/integration# ansible-playbook -i ~/hosts nxos.yaml
[DEPRECATION WARNING]: The use of 'include' for tasks has been deprecated. Use 'import_tasks' for static 
inclusions or 'include_tasks' for dynamic inclusions.
This feature will be removed in a future release. 
Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.

PLAY [nxos] ****************************************************************************************************

TASK [prepare_nxos_tests : Toggle feature nxapi - Disable] *****************************************************
ok: [evergreen-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : pause] ******************************************************************************
Pausing for 3 seconds
(ctrl+C then 'C' = continue early, ctrl+C then 'A' = abort)
ok: [dplus-nx-3.cisco.com]

TASK [prepare_nxos_tests : Toggle feature nxapi - Enable] ******************************************************
changed: [greensboro-nx-3.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : pause] ******************************************************************************
Pausing for 3 seconds
(ctrl+C then 'C' = continue early, ctrl+C then 'A' = abort)
ok: [dplus-nx-3.cisco.com]

TASK [prepare_nxos_tests : Collect interface list] *************************************************************
 [WARNING]: argument timeout has been deprecated and will be removed in a future version

ok: [dplus-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : Gather platform info] ***************************************************************
ok: [freeport-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
skipping: [dplus-nx-3.cisco.com]
skipping: [freeport-nx-3.cisco.com]
skipping: [evergreen-nx-3.cisco.com]
skipping: [greensboro-nx-3.cisco.com]
skipping: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [prepare_nxos_tests : set_fact] ***************************************************************************
skipping: [evergreen-nx-3.cisco.com]
skipping: [dplus-nx-3.cisco.com]
skipping: [greensboro-nx-3.cisco.com]
skipping: [freeport-nx-3.cisco.com]
skipping: [n9k-110.cisco.com]
skipping: [n9k-157.cisco.com]

TASK [nxos_ip_interface : collect all cli test cases] **********************************************************
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : set test_items] **********************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : run test case] ***********************************************************************
included: /root/agents-ci/ansible/test/integration/targets/nxos_ip_interface/tests/cli/sanity.yaml for dplus-nx-3.cisco.com, freeport-nx-3.cisco.com, evergreen-nx-3.cisco.com, greensboro-nx-3.cisco.com, n9k-110.cisco.com, n9k-157.cisco.com

TASK [nxos_ip_interface : debug] *******************************************************************************
ok: [evergreen-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [freeport-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [greensboro-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [dplus-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [n9k-110.cisco.com] => {
    "msg": "START TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [n9k-157.cisco.com] => {
    "msg": "START TRANSPORT:CLI nxos_ip_interface sanity test"
}

TASK [nxos_ip_interface : set_fact] ****************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : set_fact] ****************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Put interface Ethernet1/1 into default state] ****************************************
changed: [greensboro-nx-3.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Put interface Ethernet1/2 into default state] ****************************************
changed: [dplus-nx-3.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Make Ethernet1/1 a layer3 interface] *************************************************
changed: [n9k-110.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Make Ethernet1/2 a layer3 interface] *************************************************
changed: [dplus-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Ensure ipv4 address is configured] ***************************************************
changed: [dplus-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : Check Idempotence] *******************************************************************
ok: [n9k-110.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : Ensure ipv6 address is configured] ***************************************************
changed: [dplus-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : Check Idempotence] *******************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : debug] *******************************************************************************
ok: [evergreen-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [dplus-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [freeport-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [greensboro-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [n9k-110.cisco.com] => {
    "msg": "END TRANSPORT:CLI nxos_ip_interface sanity test"
}
ok: [n9k-157.cisco.com] => {
    "msg": "END TRANSPORT:CLI nxos_ip_interface sanity test"
}

TASK [nxos_ip_interface : collect all nxapi test cases] ********************************************************
ok: [greensboro-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : set test_items] **********************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : enable nxapi] ************************************************************************
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : run test case] ***********************************************************************
included: /root/agents-ci/ansible/test/integration/targets/nxos_ip_interface/tests/nxapi/sanity.yaml for dplus-nx-3.cisco.com, evergreen-nx-3.cisco.com, greensboro-nx-3.cisco.com, freeport-nx-3.cisco.com, n9k-110.cisco.com, n9k-157.cisco.com

TASK [nxos_ip_interface : debug] *******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [evergreen-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [freeport-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [greensboro-nx-3.cisco.com] => {
    "msg": "START TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [n9k-110.cisco.com] => {
    "msg": "START TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [n9k-157.cisco.com] => {
    "msg": "START TRANSPORT:NXAPI nxos_ip_interface sanity test"
}

TASK [nxos_ip_interface : set_fact] ****************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : set_fact] ****************************************************************************
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Put interface Ethernet1/1 into default state] ****************************************
changed: [dplus-nx-3.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Put interface Ethernet1/2 into default state] ****************************************
changed: [greensboro-nx-3.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Make Ethernet1/1 a layer3 interface] *************************************************
changed: [greensboro-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Make Ethernet1/2 a layer3 interface] *************************************************
changed: [greensboro-nx-3.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : Ensure ipv4 address is configured] ***************************************************
changed: [greensboro-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : Check Idempotence] *******************************************************************
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : Ensure ipv6 address is configured] ***************************************************
changed: [greensboro-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [dplus-nx-3.cisco.com]
changed: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : Check Idempotence] *******************************************************************
ok: [greensboro-nx-3.cisco.com]
ok: [n9k-110.cisco.com]
ok: [freeport-nx-3.cisco.com]
ok: [dplus-nx-3.cisco.com]
ok: [evergreen-nx-3.cisco.com]
ok: [n9k-157.cisco.com]

TASK [nxos_ip_interface : assert] ******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [greensboro-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [freeport-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [evergreen-nx-3.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-110.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}
ok: [n9k-157.cisco.com] => {
    "changed": false, 
    "failed": false, 
    "msg": "All assertions passed"
}

TASK [nxos_ip_interface : debug] *******************************************************************************
ok: [dplus-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [freeport-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [evergreen-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [greensboro-nx-3.cisco.com] => {
    "msg": "END TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [n9k-157.cisco.com] => {
    "msg": "END TRANSPORT:NXAPI nxos_ip_interface sanity test"
}
ok: [n9k-110.cisco.com] => {
    "msg": "END TRANSPORT:NXAPI nxos_ip_interface sanity test"
}

TASK [nxos_ip_interface : disable nxapi] ***********************************************************************
changed: [dplus-nx-3.cisco.com]
changed: [evergreen-nx-3.cisco.com]
changed: [freeport-nx-3.cisco.com]
changed: [greensboro-nx-3.cisco.com]
changed: [n9k-110.cisco.com]
changed: [n9k-157.cisco.com]

PLAY RECAP *****************************************************************************************************
dplus-nx-3.cisco.com       : ok=52   changed=14   unreachable=0    failed=0   
evergreen-nx-3.cisco.com   : ok=50   changed=14   unreachable=0    failed=0   
freeport-nx-3.cisco.com    : ok=50   changed=14   unreachable=0    failed=0   
greensboro-nx-3.cisco.com  : ok=50   changed=14   unreachable=0    failed=0   
n9k-110.cisco.com          : ok=50   changed=14   unreachable=0    failed=0   
n9k-157.cisco.com          : ok=51   changed=14   unreachable=0    failed=0   

root@agent-lab15-pm:~/agents-ci/ansible/test/integration#